src/core/updateWebpackConfig.js

```diff
@@ -13,31 +13,25 @@ const updateWebpackConfig = (webpackConfig, env, serverInfo) => {
     {
       test: /\.jsx?$/,
       include: srcFolder,
-      loader: 'babel',
+      loader: 'babel-loader',
       query: {
+        babelrc: false,
         presets: [
           'es2015',
           'react',
         ],
```

Comment:
Thanks @sapegin, but I'm not sure I get why:
> Relying on loaders will cause issues.

Can you please briefly explain me what's going to be the problem?